.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Isn't this ignore specific to your editor you are using?  If so, that should not be added to this repo and instead you should perhaps add to your global git ignore